pkg/morph/client/constructor.go

```diff
@@ -71,10 +71,10 @@ func defaultConfig() *cfg {
 	}
 }
 
-// ErrStaleNode is returned from [New] when minimal required height
+// ErrStaleNodes is returned from [New] when minimal required height
 // requirement specified in [WithMinRequiredBlockHeight] is not
 // satisfied by the given nodes.
-var ErrStaleNode = errors.New("RPC node is not yet up to date")
+var ErrStaleNodes = errors.New("RPC nodes are not yet up to date")
```

Comment:
Why this change, really?

Reply:
cause now it is retuned only if _every_ node is _not_ up to date. calling `SwitchRPC` outside of the morph pkg is strange to me